main_spec.tex

```diff
@@ -103,10 +101,16 @@ \subsubsection{\textbf{SHPCLMOVE}}\label{subsec:shpclmove}
 \subsubsection{\textbf{SHPDEALLOC}}\label{subsec:shpdealloc}
 \input{content/shpdealloc.tex}
 
+\subsection{Thread Support}
```

Comment:
Why is this section ordered as 8.3 after Memory Management and not after Library Setup? I would expect Thread Support to be section 8.2 (section-adjacent to `shmem_init`).